.woodpecker.yml

```diff
@@ -1,7 +1,43 @@
 variables:
-  - &rust_image "rustlang/rust:nightly"
+  - &rust_image "rust:latest"
```

Comment:
You can just use `rust` then, since its just the stable tag.

Is there any reason you can't run nightly tho?

Reply:
just matching how it is currently compiled in dev

is there a lot of benefits with nightly?